### PR TITLE
Re-write printing in logfmt format in Ballerina

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Purpose
$Subject
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/1592

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: log-ballerina/natives.bal [hunk 1]
```diff
@@ -159,7 +159,52 @@ isolated function stderrStream() returns handle = @java:FieldGet {
...
+    temp = escapseExternal(temp, java:fromString("\t"), java:fromString("\\t"));
+    temp = escapseExternal(temp, java:fromString("\n"), java:fromString("\\n"));
+    temp = escapseExternal(temp, java:fromString("\r"), java:fromString("\\r"));
+    temp = escapseExternal(temp, java:fromString("'"), java:fromString("\\'"));
+    temp = escapseExternal(temp, java:fromString("\""), java:fromString("\\\""));
+    string? updatedString = java:toString(temp);
+    return updatedString.toBalString();
+}
+
+public isolated function escapseExternal(handle receiver, handle target, handle replacement) returns handle = @java:Method {
```
Comment (daneshk) [inline log-ballerina/natives.bal:204]: shall we make this function private. and also function is not correct, should be like `replaceString` or something

Comment (MadhukaHarith92) [reply log-ballerina/natives.bal]: Fixed in 04c5723

File: log-ballerina/natives.bal [hunk 2]
```diff
@@ -159,7 +159,52 @@ isolated function stderrStream() returns handle = @java:FieldGet {
     'class: "java/lang/System"
 } external;
 
-isolated function printLogFmtExtern(LogRecord logRecord) returns string = @java:Method {'class: "org.ballerinalang.stdlib.log.Utils"} external;
+isolated function printLogFmtExtern(LogRecord logRecord) returns string {
```
Comment (daneshk) [inline log-ballerina/natives.bal:162]:
```suggestion
isolated function printLogFmt(LogRecord logRecord) returns string {
```

Comment (daneshk) [reply log-ballerina/natives.bal:162]: shall we remove extern part from the function name.

Comment (MadhukaHarith92) [reply log-ballerina/natives.bal]: Updated in ce2e747

**Review by daneshk — APPROVED** [review]
LGTM, Let's merge this after beta2 is released

**Pull request merged**